ra/ra.go

```diff
@@ -963,9 +963,14 @@ func (ra *RegistrationAuthorityImpl) UpdateAuthorization(ctx context.Context, ba
 
 	// Store the updated version
 	if err = ra.SA.UpdatePendingAuthorization(ctx, authz); err != nil {
-		// This can pretty much only happen when the client corrupts the Challenge
-		// data.
-		err = core.MalformedRequestError("Challenge data was corrupted")
+		// Previously this was a MalformedREquestError alongside a comment explaning
```

Comment:
:+1: Makes sense - the history is there in git for the curious archaeologist and it isn't very interesting.